docs/translations/api-docs/date-pickers/date-field/date-field.json

```diff
@@ -17,6 +17,9 @@
     "disableFuture": {
       "description": "If <code>true</code>, disable values after the current date for date components, time for time components and both for date time components."
     },
+    "disableNonContiguousRanges": {
+      "description": "If <code>true</code>, doesn&#39;t throws an error when a range containing disabled dates is selected."
```

Comment:
Notice that The description of the new prop on the api-docs is different for `Fields`, `DateRangeCalendar` and `Pickers`, since the behavior is different depending on the component !